automate_url_open.md

```diff
@@ -0,0 +1,44 @@
+### Opening custom URLs via Custom Buttons and Automate
+
+For a number of different entities users can create custom [toolbar
+buttons](ui/toolbars.md) that run an automate method on the particular entity,
+let that automate method calculate an URL and then have the UI open such URL in
+a new windows.
+
+#### Steps
+
+Create custom button with Automate method:
+
+1) Automation -> Automate -> Explorer:
+
+   a) create a new domain with a name
+
+   b) select ManageIQ/System/Request
+
+   c) Configuration -> Copy this Class (copy it to your new domain from step 1a)
+
+   d) select Request Class in your new domain -> Configuration -> Add a new Instance
+      * name `TestOpenUrl`
+      * to `meth1` write `test_open_url`
+
+   e) Select Request class in your new domain, click Methods -> Configuration -> Add a new Method
+      * type: inline
+      * name: test_open_url
+      * data should be (or anything that sets `vm.remote_console_url` to something):
+
+```
+      vm = $evm.root['vm']
+      $evm.log(:info, "VM to launch SSH for is #{vm.hostnames[0]}")
+      vm.external_url = "https://www.google.com"
+
+```
+*important: use URL including the protocol (https) or the browser will ignore the request*
+*also disable URL pop-up blocking in your browser to make this work*
```

Comment:
I'm afraid that visually, both these sentences will be merged into one (that will not look OK).

Reply:
yes, see here: https://github.com/martinpovolny/guides/blob/automate_url_opener/automate_url_open.md